src/server/search/search_family.cc

```diff
@@ -252,19 +252,25 @@ optional<AggregateParams> ParseAggregatorParamsOrReply(CmdArgParser parser,
   AggregateParams params;
   tie(params.index, params.query) = parser.Next<string_view, string_view>();
 
-  while (parser.HasNext()) {
-    // LOAD count field [field ...]
-    if (parser.Check("LOAD")) {
-      size_t num_fields = parser.Next<size_t>();
+  /*
+  Parse LOAD count field [field ...]
+
+  LOAD options are at the beginning of the query, so we need to parse them first
+  */
```

Comment:
```suggestion
  // Parse LOAD count field [field ...]
  // LOAD options are at the beginning of the query, so we need to parse them first
```
nit: It takes less vertical space like that :)